tools/serverpod_cli/lib/src/create/database_setup.dart

```diff
@@ -23,12 +24,38 @@ class DatabaseSetup {
       projectName: name,
     );
 
-    var migration = await generator.createMigration(
-      force: false,
-      config: config,
-    );
+    MigrationVersion? migration;
+    try {
+      migration = await generator.createMigration(
+        force: false,
+        config: config,
+      );
+    } on MigrationVersionLoadException catch (e) {
+      log.error(
+        'Unable to determine latest database definition due to a corrupted '
+        'migration. Please re-create or remove the migration version and try '
+        'again. Migration version: "${e.versionName}".',
+      );
+      log.error(e.exception);
+    } on GenerateMigrationDatabaseDefinitionException {
+      log.error('Unable to generate database definition for project.');
```

Comment:
Question: Do we know more about what went wrong? If I would encounter this error it wouldn't really help me moving forward.